src/common/PeopleList/index.js

```diff
@@ -1,19 +1,38 @@
+import React from "react";
 import { Main } from "../Main";
 import MainHeader from "../MainHeader";
 import { List, ListItem, StyledLink } from "./styled";
 import PersonTile from "../PersonTile";
+import { usePopularPeopleData } from "../../features/MoviesBrowser/PopularPeople/usePopularPeopleData";
+import Loading from "../Loading";
 
-const PeopleList = ({ title, picture, name, role, increasedGap }) => (
-  <Main>
-    <MainHeader title={title} />
-    <List $increasedGap={increasedGap}>
-      <ListItem>
-        <StyledLink>
-          <PersonTile picture={picture} name={name} role={role} />
-        </StyledLink>
-      </ListItem>
-    </List>
-  </Main>
-);
+const PeopleList = ({ title, increasedGap }) => {
+  const popularPeopleData = usePopularPeopleData();
+
```

Comment:
I think it would've been better if PeopleList would receive passed props from other components like PopularPeople. It's supposed to be reusable component, not PopularPeople itself.. 😅

Reply:
Done :-)

Reply:
From what I'm seeing PeopleList is still using data from `usePopularPeopleData`, not from passed data prop.. 😅

Reply:
Done :-D

Reply:
Nice, that's exactly what I meant 😎